modVM/src/org/aion/vm/BulkExecutor.java

```diff
@@ -157,45 +156,47 @@ public BulkExecutor (
                         this.executionBatch.getTransactions().get(currentIndex);
 
                 KernelInterface vmKernel;
-                if (transactionIsForFastVirtualMachine(firstTransactionInNextBatch)) {
+                if (transactionIsForAionVirtualMachine(firstTransactionInNextBatch)) {
                     vmKernel =
-                            new KernelInterfaceForFastVM(
+                            new KernelInterfaceForAVM(
                                     this.repositoryChild.startTracking(),
                                     this.allowNonceIncrement,
                                     this.isLocalCall);
                     virtualMachineForNextBatch =
-                            VirtualMachineProvider.getVirtualMachineInstance(VM.FVM, vmKernel);
+                            VirtualMachineProvider.getVirtualMachineInstance(VM.AVM, vmKernel);
                     nextBatchToExecute =
-                            fetchNextBatchOfTransactionsForFastVirtualMachine(currentIndex);
+                            fetchNextBatchOfTransactionsForAionVirtualMachine(currentIndex);
                 } else {
                     vmKernel =
-                            new KernelInterfaceForAVM(
+                            new KernelInterfaceForFastVM(
                                     this.repositoryChild.startTracking(),
                                     this.allowNonceIncrement,
                                     this.isLocalCall);
                     virtualMachineForNextBatch =
-                            VirtualMachineProvider.getVirtualMachineInstance(VM.AVM, vmKernel);
+                            VirtualMachineProvider.getVirtualMachineInstance(VM.FVM, vmKernel);
                     nextBatchToExecute =
-                            fetchNextBatchOfTransactionsForAionVirtualMachine(currentIndex);
+                            fetchNextBatchOfTransactionsForFastVirtualMachine(currentIndex);
```

Comment:
reason for reversing avm fvm order in the if-else condition

Reply:
Just seems more natural to do `if (condition)` instead of `if (!condition)`